From: Julian Andres Klode <julian.klode@canonical.com>
Date: Thu, 25 Jan 2018 12:02:55 +0100
Subject: fix-usrmerge-paths

Fix paths to use /lib/udev instead of /usr/lib/udev, since we have not
done a usrmerge yet.

Bug-Debian: https://bugs.debian.org/886938
Forwarded: no
Origin: vendor
---
 multipath/11-dm-mpath.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/multipath/11-dm-mpath.rules
+++ b/multipath/11-dm-mpath.rules
@@ -101,7 +101,7 @@
 
 # kpartx_id is very robust, it works for suspended maps and maps
 # with 0 dependencies. It sets DM_TYPE, DM_PART, DM_WWN
-TEST=="/usr/lib/udev/kpartx_id", \
+TEST=="/lib/udev/kpartx_id", \
 	IMPORT{program}=="kpartx_id %M %m $env{DM_UUID}"
 
 ENV{DM_TYPE}=="?*", ENV{DM_SERIAL}=="?*", \
